### PR TITLE
[DOCFIX] Improve quick start guide

### PR DESCRIPTION
**Pull request by dbw9580**
Minor tweaks to the quick start guide.

### CODE REVIEW

File: docs/en/overview/Getting-Started.md [hunk 1]
```diff
@@ -23,13 +23,8 @@ The guide will cover the following tasks:
...
 **Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
-
-If you are trying to speedup SQL analytics, you can try the Presto Alluxio Getting Started tutorial:
-
-<p align="center">
-<a href="https://www.alluxio.io/alluxio-presto-sandbox-docker/">
- <img src="https://www.alluxio.io/app/uploads/2019/07/laptop-docker.png" width="250" alt="Laptop with Docker"/></a>
-</p>
+If you are trying to speedup SQL analytics, you can try the 
+[Presto Alluxio Getting Started](https://www.alluxio.io/alluxio-presto-sandbox-docker/) tutorial.
```
Comment (dbw9580) [inline docs/en/overview/Getting-Started.md:27]: The picture just says `on laptop with Docker`, which is confusing in this context. We had multiple options here that allowed people to select different scenarios to get started, so the pictures were good to have. Now there is only one option to start with Presto, so the picture is redundant.

**Review by Xenorith — APPROVED** [review]
appreciate the thoughtful changes you made here to make this more user friendly

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**